homeassistant/components/hunterdouglas_powerview/entity.py

```diff
@@ -117,3 +117,8 @@ def device_info(self) -> DeviceInfo:
         device_info[ATTR_SW_VERSION] = sw_version
 
         return device_info
+
+    async def async_update(self) -> None:
+        """Refresh shade position."""
+        await self._shade.refresh()
+        self.data.update_shade_positions(self._shade.raw_data)
```

Comment:
Is this different than the default coordinator update entity method that refreshes the coordinator data?

Reply:
the default update method just pulls the data that the hub has in memory - this method will force the mose recent information (poll the shade for an update)
This is not used as default as battery shades would be drained too quickly
Hope that answers the question

Reply:
The data in the hub and the coordinator SHOULD be the same. This is actually telling the hub ‘we don’t trust what you’re telling me, check with the shade.

This method polls the shade for updated position data - so the hub may say it is at 70%, this method will poll the shade for its true position which may be 20% for instance if someone has moved the shade already

As a move via pebble remotes won’t refresh position in the hub.